CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Added
+
+* Tracing: Added support for W3C Baggage extraction and injection by default.
+* Tracing: Adds api method to access and modify baggage object.
```

Comment:
No need to add these to this file. The section "Change log entry" in the PR description will make it here during release.